gtda/homology/cubical.py

```diff
@@ -51,6 +58,10 @@ class CubicalPersistence(BaseEstimator, TransformerMixin, PlotterMixin):
         filtration value ``numpy.inf``. ``None`` assigns the maximum pixel
         values within all images passed to :meth:`fit`.
 
+    reduced_homology : bool, optional, default: ``True``
+       If ``True``, one feature in homology dimension 0 with infinite death is
```

Comment:
I think I would avoid the term feature here. 

`If ``True``, a persistent point in homology dimension 0 with infinte death is discarded in all output diagrams.`

Reply:
This is also valid for all homology transformers.

Reply:
Ok, except I would use "triple" or other term that has been used elsewhere in the docstring.

Reply:
Yes, that's better!

Reply:
New wording:
```
       If ``True``, the earliest-born triple in homology dimension 0 which has
       infinite death is discarded in :meth:`transform`.
```
It also stresses that it is not just any old triple in dim 0, but the first-born.

Reply:
I would say: "in each diagram". It's obvious but it may confuse early users otherwise.

Reply:
Good point!